Best Practices/Challenges/DI/DI.Challenge/DIChallenge.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DI.Challenge.Services;
+
+namespace DI.Challenge
+{
+    public class DIChallenge
+    {
+        private readonly TransientService _transientService;
+        private readonly ScopedService _scopedService;
+        private readonly SingletonService _singletonService;
+
+        public DIChallenge()
+        {
+            _transientService = new TransientService();
```

Comment:
Suggestion: I wonder if it's worth having these already in the constructor, so they don't need to uncomment any code in the test files